### PR TITLE
Add job to map tenant source and target from the control-api, through the billing prometheus

### PR DESCRIPTION
**Pull request by bastjan**
Set to dry-run per default for now.

## Checklist

- [x] The PR has a meaningful title. It will be used to auto generate the
      changelog.
      The PR has a meaningful description that sums up the change. It will be
      linked in the changelog.
- [x] PR contains a single logical change (to build a better changelog).
- [x] Update the documentation.
- [x] Categorize the PR by adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.

### CODE REVIEW

File: component/main.jsonnet [hunk 1]
```diff
@@ -137,6 +137,31 @@ local syncCategoriesContainer = {
...
+    checkMigrationContainer,
+  ],
+  containers: [
+    {
+      name: 'mapping',
+      image: formatImage(params.images.reporting),
+      env+: dbEnv + promEnv,
+      command: [ 'sh', '-c' ],
+      args: [
+        'appuio-cloud-reporting tenantmapping --begin=$(date -d "now -3 hours" -u +"%Y-%m-%dT%H:00:00Z") --repeat-until=$(date -u -Iseconds)'
```
Comment (simu) [inline component/main.jsonnet:151]: What's the reasoning for doing this for the last three hours once an hour? Is this just to make sure we don't miss anything?

side-note: I see that we do the same thing for the `check-missing` cronjob, which also runs once an hour.

**Review by simu — APPROVED** [review]
LGTM

File: component/main.jsonnet (hunk 1 above)
Comment (bastjan) [reply component/main.jsonnet:151]: Because of the varying lag of Thanos getting updated. So we don't miss anything if Thanos is not yet updated.

**Review by HappyTetrahedron — APPROVED** [review]
LGTM

**Comment by HappyTetrahedron** [comment]
I just merged https://github.com/appuio/component-appuio-cloud-reporting/pull/59 -- you probably have to merge in master and regenerate golden tests now

**Pull request merged**